corehq/apps/change_feed/partitioners.py

```diff
@@ -14,26 +13,6 @@
 }
 
 
-def choose_best_partition_for_topic(topic):
-    if topic not in _get_topic_to_pillow_map():
-        # None means there's no best, use the default
-        return None
-
-    try:
-        backlog_lengths_by_partition = _get_backlog_lengths_by_partition(topic)
```

Comment:
I bet there's more to delete here. https://github.com/dimagi/commcare-hq/pull/26747/files is where most of this was added, so maybe it can be a good guide for what else to delete.

Reply:
Yep just realized this, deleted the entire file, amended the commit, and force pushed (was hoping to push before anyone reviewed 😄)